### PR TITLE
[zh-cn] Translate /docs/concepts/security/multi-tenancy/ to Chinese #…

### PR DESCRIPTION
**Pull request by javadoors**
[zh-cn] Translate /docs/concepts/security/multi-tenancy/ to Chinese #34798

Add page： /zh-cn/docs/concepts/security/multi-tenancy/

### CODE REVIEW

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 1]
```diff
@@ -0,0 +1,1115 @@
...
+<!--
+Control plane virtualization allows for isolation of non-namespaced resources 
+at the cost of somewhat higher resource usage and more difficult cross-tenant sharing. 
+It is a good option when namespace isolation is insufficient but dedicated clusters are undesirable, 
+due to the high cost of maintaining them (especially on-prem) 
+or due to their higher overhead and lack of resource sharing. 
+However, even within a virtualized control plane, 
+you will likely see benefits by using namespaces as well.
+-->
+控制平面虚拟化允许以更高的资源使用率和更困难的跨租户共享为代价隔离非命名空间资源。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:916]:
```suggestion
控制平面虚拟化允许以更高的资源使用率和更困难的跨租户共享为代价隔离非命名空间作用域的资源。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 2]
```diff
@@ -0,0 +1,1115 @@
...
+当命名空间隔离不足但不希望使用专用集群时，这是一个不错的选择，
+因为维护它们的成本很高（尤其是本地集群），
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:918]:
```suggestion
因为维护专用集群的成本很高（尤其是本地集群），
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 3]
```diff
@@ -0,0 +1,1115 @@
...
+或者由于它们的开销较高且缺乏资源共享。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:919]:
```suggestion
或者由于专用集群的额外开销较高且缺乏资源共享。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 4]
```diff
@@ -0,0 +1,1115 @@
...
+
+<!--
+The two options are discussed in more detail in the following sections:
+-->
+以下各节将更详细地讨论这两个选项：
+
+<!--
+### Namespace per tenant
+-->
+### 每个租户的命名空间 {#namespace-per-tenant}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:930]:
```suggestion
### 每个租户独立的命名空间 {#namespace-per-tenant}
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 5]
```diff
@@ -0,0 +1,1115 @@
...
+This ensures that each workload only has access to its own resources, such as Config Maps and Secrets, 
+and allows you to tailor dedicated security policies for each workload. 
+In addition, it is a best practice to give each namespace names that are unique across your entire fleet 
+(i.e., even if they are in separate clusters), 
+as this gives you the flexibility to switch between dedicated and shared clusters in the future, 
+or to use multi-cluster tooling such as service meshes.
+-->
+如前所述，你应该考虑将每个工作负载隔离在其自己的命名空间中，
+即使你使用的是专用集群或虚拟化控制平面。
+这可确保每个工作负载只能访问其自己的资源，例如 Config Map 和 Secret，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:944]:
```suggestion
这可确保每个工作负载只能访问其自己的资源，例如 ConfigMap 和 Secret，
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 6]
```diff
@@ -0,0 +1,1115 @@
...
+not just the workload level, 
+since there are often policies that apply to all workloads owned by a single tenant. 
+However, this raises its own problems. 
+Firstly, this makes it difficult or impossible to customize policies to individual workloads, 
+and secondly, it may be challenging to come up with a single level of "tenancy" that should be given a namespace.  
+For example, an organization may have divisions, teams, and subteams - which should be assigned a namespace?
+-->
+相反，在租户级别分配命名空间也有优势，
+而不仅仅是工作负载级别，
+因为通常有适用于单个租户拥有的所有工作负载的策略。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:961]:
```suggestion
因为通常有一些策略适用于单个租户拥有的所有工作负载。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 7]
```diff
@@ -0,0 +1,1115 @@
...
+然而，这也带来了自己的问题。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:962]:
```suggestion
然而，这种方案也有自己的问题。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 8]
```diff
@@ -0,0 +1,1115 @@
...
+首先，这使得为单个工作负载定制策略变得困难或不可能，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:963]:
```suggestion
首先，这使得为各个工作负载定制策略变得困难或不可能，
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 9]
```diff
@@ -0,0 +1,1115 @@
...
+其次，提出应该赋予命名空间的单一级别的“租户”可能具有挑战性。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:964]:
```suggestion
其次，确定应该赋予命名空间的单一级别的 “租户” 可能很困难。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 10]
```diff
@@ -0,0 +1,1115 @@
...
+which allows you to organize your namespaces into hierarchies, 
+and share certain policies and resources between them. 
+It also helps you manage namespace labels, namespace lifecycles, and delegated management, 
+and share resource quotas across related namespaces. 
+These capabilities can be useful in both multi-team and multi-customer scenarios.
+-->
+为了解决这个问题，Kubernetes 提供了
+[Hierarchical Namespace Controller (HNC)](https://github.com/kubernetes-sigs/hierarchical-namespaces)，
+它允许你将命名空间组织成层次结构，
+并在它们之间共享某些策略和资源。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:979]:
```suggestion
它允许你将多个命名空间组织成层次结构，并在它们之间共享某些策略和资源。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 11]
```diff
@@ -0,0 +1,1115 @@
...
+* [Kyverno](https://kyverno.io/)
+* [OPA/Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
+-->
+* [Kyverno](https://kyverno.io/)
+* [OPA/Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
+
+<!--
+### Virtual control plane per tenant
+-->
+### 每个租户的虚拟控制平面 {#virtual-control-plane-per-tenant}
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1031]:
```suggestion
### 每个租户独立的虚拟控制面    {#virtual-control-plane-per-tenant}
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 12]
```diff
@@ -0,0 +1,1115 @@
...
+
+<!--
+Another form of control-plane isolation is to use Kubernetes extensions to 
+provide each tenant a virtual control-plane that enables segmentation of cluster-wide API resources. 
+[Data plane isolation](#data-plane-isolation) techniques can be used 
+with this model to securely manage worker nodes across tenants.
+-->
+控制平面隔离的另一种形式是使用 Kubernetes 扩展为每个租户提供一个虚拟控制平面，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1039]:
```suggestion
控制面隔离的另一种形式是使用 Kubernetes 扩展为每个租户提供一个虚拟控制面，
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 13]
```diff
@@ -0,0 +1,1115 @@
...
+以实现集群范围内 API 资源的分段。
+[数据平面隔离](#data-plane-isolation)技术可以与此模型一起使用，
+以安全地管理跨租户的工作节点。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1042]:
```suggestion
以安全地跨多个租户管理工作节点。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 14]
```diff
@@ -0,0 +1,1115 @@
...
+The virtual control plane based multi-tenancy model extends namespace-based multi-tenancy 
+by providing each tenant with dedicated control plane components, 
+and hence complete control over cluster-wide resources and add-on services. 
+Worker nodes are shared across all tenants, 
+and are managed by a Kubernetes cluster that is normally inaccessible to tenants. 
+This cluster is often referred to as a _super-cluster_ (or sometimes as a _host-cluster_). 
+Since a tenant’s control-plane is not directly associated 
+with underlying compute resources it is referred to as a _virtual control plane_.
+-->
+基于虚拟控制平面的多租户模型通过为每个租户提供专用控制平面组件来扩展基于命名空间的多租户，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1054]:
```suggestion
基于虚拟控制面的多租户模型通过为每个租户提供专用控制面组件来扩展基于命名空间的多租户，
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 15]
```diff
@@ -0,0 +1,1115 @@
...
+从而完全控制集群范围的资源和附加服务。
+工作节点在所有租户之间共享，
+并由租户通常无法访问的 Kubernetes 集群管理。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1057]:
```suggestion
工作节点在所有租户之间共享，并由租户通常无法访问的 Kubernetes 集群管理。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 16]
```diff
@@ -0,0 +1,1115 @@
...
+该集群通常被称为 **super-cluster**（或有时称为 **host-cluster**）。
+由于租户的控制平面不直接与底层计算资源相关联，
+因此它被称为**虚拟控制平面**。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1060]:
```suggestion
由于租户的控制面不直接与底层计算资源相关联，因此它被称为**虚拟控制平面**。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 17]
```diff
@@ -0,0 +1,1115 @@
...
+
+<!--
+A virtual control plane typically consists of the Kubernetes API server, 
+the controller manager, and the etcd data store. 
+It interacts with the super cluster via a metadata synchronization controller 
+which coordinates changes across tenant control planes and the control plane of the super--cluster. 
+-->
+虚拟控制平面通常由 Kubernetes API 服务器、控制器管理器和 etcd 数据存储组成。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1068]:
```suggestion
虚拟控制面通常由 Kubernetes API 服务器、控制器管理器和 etcd 数据存储组成。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 18]
```diff
@@ -0,0 +1,1115 @@
...
+and are managed by a Kubernetes cluster that is normally inaccessible to tenants. 
+This cluster is often referred to as a _super-cluster_ (or sometimes as a _host-cluster_). 
+Since a tenant’s control-plane is not directly associated 
+with underlying compute resources it is referred to as a _virtual control plane_.
+-->
+基于虚拟控制平面的多租户模型通过为每个租户提供专用控制平面组件来扩展基于命名空间的多租户，
+从而完全控制集群范围的资源和附加服务。
+工作节点在所有租户之间共享，
+并由租户通常无法访问的 Kubernetes 集群管理。
+该集群通常被称为 **super-cluster**（或有时称为 **host-cluster**）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1058]:
```suggestion
该集群通常被称为 **超集群（Super-Cluster）**（或有时称为 **host-cluster**）。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 19]
```diff
@@ -0,0 +1,1115 @@
...
+因此它被称为**虚拟控制平面**。
+
+<!--
+A virtual control plane typically consists of the Kubernetes API server, 
+the controller manager, and the etcd data store. 
+It interacts with the super cluster via a metadata synchronization controller 
+which coordinates changes across tenant control planes and the control plane of the super--cluster. 
+-->
+虚拟控制平面通常由 Kubernetes API 服务器、控制器管理器和 etcd 数据存储组成。
+它通过元数据同步控制器与超级集群交互，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1069]:
```suggestion
它通过元数据同步控制器与超集群交互，
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 20]
```diff
@@ -0,0 +1,1115 @@
...
+该控制器协调跨租户控制平面和超级集群控制平面的变化。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1070]:
```suggestion
该控制器跨租户控制面和超集群控制面对变化进行协调。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 21]
```diff
@@ -0,0 +1,1115 @@
...
+By using per-tenant dedicated control planes, 
+most of the isolation problems due to sharing one API server among all tenants are solved. 
+Examples include noisy neighbors in the control plane, 
+uncontrollable blast radius of policy misconfigurations, 
+and conflicts between cluster scope objects such as webhooks and CRDs.  
+Hence, the virtual control plane model is particularly suitable for cases 
+where each tenant requires access to a Kubernetes API server and expects the full cluster manageability. 
+-->
+通过使用每个租户的专用控制平面，
+解决了由于所有租户共享一个 API 服务器而导致的大部分隔离问题。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1082]:
```suggestion
通过使用每个租户单独的专用控制面，可以解决由于所有租户共享一个 API 服务器而导致的大部分隔离问题。
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 22]
```diff
@@ -0,0 +1,1115 @@
...
+
+<!--
+The improved isolation comes at the cost of running 
+and maintaining an individual virtual control plane per tenant. 
+In addition, per-tenant control planes do not solve isolation problems in the data plane, 
+such as node-level noisy neighbors or security threats. 
+These must still be addressed separately.
+-->
+改进的隔离是以每个租户运行和维护一个单独的虚拟控制平面为代价的。
+此外，每租户控制平面不能解决数据平面中的隔离问题，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/security/multi-tenancy.md:1096]:
```suggestion
此外，租户层面的控制面不能解决数据面的隔离问题，
```

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 23]
```diff
@@ -0,0 +1,1115 @@
...
+-->
+### 存储隔离 {#storage-isolation}
+
+<!--
+Kubernetes offers several types of volumes that can be used as persistent storage for workloads. 
+For security and data-isolation, [dynamic volume provisioning](/docs/concepts/storage/dynamic-provisioning/) 
+is recommended and volume types that use node resources should be avoided. 
+-->
+Kubernetes 提供了几种类型的卷，可以用作工作负载的持久存储。
+为了安全和数据隔离，建议使用[动态卷供应](/zh-cn/docs/concepts/storage/dynamic-provisioning/)，
```
[the remark this answers is not in the document]
Comment (javadoors) [reply content/zh-cn/docs/concepts/security/multi-tenancy.md:515]: This page /zh-cn/docs/concepts/storage/dynamic-provisioning/ translate is "供应"

File: content/zh-cn/docs/concepts/security/multi-tenancy.md [hunk 24]
```diff
@@ -0,0 +1,1115 @@
...
+
+<!--
+Pods can request storage using a [PersistentVolumeClaim](/docs/concepts/storage/persistent-volumes/). 
+A PersistentVolumeClaim is a namespaced resource, 
+which enables isolating portions of the storage system 
+and dedicating it to tenants within the shared Kubernetes cluster. 
+However, it is important to note that a PersistentVolume is a cluster-wide resource 
+and has a lifecycle independent of workloads and namespaces.
+-->
+Pod 可以使用[持久卷](/zh-cn/docs/concepts/storage/persistent-volumes/)请求存储。
```
[the remark this answers is not in the document]
Comment (javadoors) [reply content/zh-cn/docs/concepts/security/multi-tenancy.md:534]: 中文链接我都参考了已经存在的翻译。

File: content/zh-cn/docs/concepts/security/multi-tenancy.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (tengqm) [reply content/zh-cn/docs/concepts/security/multi-tenancy.md:515]: 那个是早期翻译，可以改掉。

**Comment by javadoors** [comment]
pls review again,thanks.

**Comment by tengqm** [comment]
@javadoors Please mark conversations resolved if you agreed to the review comments and you have modified the page as suggested. Only the PR author can mark them as resolved. As an author, you don't want the reviewers to double check each and every change, right?

**Comment by javadoors** [comment]
> @javadoors Please mark conversations resolved if you agreed to the review comments and you have modified the page as suggested. Only the PR author can mark them as resolved. As an author, you don't want the reviewers to double check each and every change, right?

ok,I will mark it.-_-

**Pull request merged**